internal/integration_test/fuzzcases/fuzzcases_test.go

```diff
@@ -298,3 +298,35 @@ func Test730(t *testing.T) {
 		}
 	})
 }
+
+func Test733(t *testing.T) {
+	run(t, func(t *testing.T, r wazero.Runtime) {
+		mod, err := r.InstantiateModuleFromBinary(ctx, getWasmBinary(t, 733))
+		require.NoError(t, err)
+
+		name := "out of bounds"
+		t.Run(name, func(t *testing.T) {
+			f := mod.ExportedFunction(name)
+			require.NotNil(t, f)
+			_, err = f.Call(ctx)
+			require.Error(t, err)
+			require.Contains(t, err.Error(), "out of bounds memory")
+		})
+
+		// Note: this case uses large memory space, so can be slow.
```

Comment:
clarify how slow in some unit (as example) so we know slow like 10s or slow like 10m